src/components/ProjectSearch/TextSearch.js

```diff
@@ -59,7 +59,7 @@ export default class TextSearch extends Component {
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
     if (this.refs.searchInput) {
-      this.refs.searchInput.refs.input.focus();
+      this.refs.searchInput.$input.focus();
```

Comment:
hmm this looks like it might give us trouble later on -- we are controlling a child component from a parent component; this makes it difficult to separate them -- we probably want searchInput to do this, rather than doing it here.